yoti-sdk-spring-boot-example/README.md

```diff
@@ -21,9 +21,9 @@ Before you start, you'll need to create an Application in [Dashboard](https://ww
 ```
 
 ## Building your example server-app
-1. In Dashboard edit the "Callback URL" field with the URL pointing to your /login endpoint.
-`Ex. https://mydomain.com/login`. Note that your endpoint must be accessible on the Internet.
-1. Edit the `resources/application.yml` and replace the `yoti-client-sdk-id-from-dashboard` value with the `Yoti client SDK ID` you can find in Dashboard.
+1. In the [Yoti Dashboard](https://www.yoti.com/dashboard/applications) set the application domain of your app to `https://localhost:8443/`. Note that your endpoint must be accessible on the Internet.
```

Comment:
It is not necessary that it is accessible from the Internet, as long as it is accessible from the machine that is visualising the QR code.

Reply:
`Note that your endpoint must be accessible from the machine that is visualising the QR code`?

Reply:
You mean, the machine that is _displaying_ the QR code?

Reply:
Yes that makes more sense, will change to that